index.js

```diff
@@ -11,6 +11,38 @@ const pAccess = promisify(fs.access);
 // Path to included `xdg-open`.
 const localXdgOpenPath = path.join(__dirname, 'xdg-open');
 
+/**
+Get the mount point for fixed drives in WSL.
+
+@inner
+@returns {string} The mount point.
+*/
+const getWslDrivesMountPoint = (() => {
```

Comment:
The `open` function is async, so it should not contain any synchronous IO. That would be unexpected for consumers. You can use `util.promisify` on the `fs` methods.

Reply:
I made this function synchronous according to the previous review (see [comment](https://github.com/sindresorhus/open/pull/219#discussion_r570356185)).